scipy/signal/_filter_design.py

```diff
@@ -5341,11 +5340,13 @@ def iircomb(w0, Q, ftype='notch', fs=2.0, *, pass_zero=False):
         G0, G = 1, 0
     elif ftype == 'peak':
         G0, G = 0, 1
-    GB = 1 / np.sqrt(2)
 
     # Compute beta
     # Eq. 11.5.3 (p. 591) from reference [1]
-    beta = np.sqrt((GB**2 - G0**2) / (G**2 - GB**2)) * np.tan(N * w_delta / 4)
+    # Note that formulas can be further simplified when -3dB attenuation value
+    # GB = 1 / np.sqrt(2):
+    #   np.sqrt((GB**2 - G0**2) / (G**2 - GB**2)) = 1
```

Comment:
The wording in this comment could also be made a little bit clearer, e.g. something like:

```python
# Due to assuming a -3 dB attenuation value, i.e, assuming GB = 1 / np.sqrt(2), 
# the following term of Eq. 11.5.3 simplifies to:
#   np.sqrt((GB**2 - G0**2) / (G**2 - GB**2)) = 1
```

Reply:
> The wording in this comment could also be made a little bit clearer, e.g. something like:
> 
> ```python
> # Due to assuming a -3 dB attenuation value, i.e, assuming GB = 1 / np.sqrt(2), 
> # the following term of Eq. 11.5.3 simplifies to:
> #   np.sqrt((GB**2 - G0**2) / (G**2 - GB**2)) = 1
> ```

Same here, I prefer your version, I will apply your changes.